opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -87,13 +95,15 @@ def __init__(
         self.out = out
         self.formatter = formatter
 
-    def export(self, metrics: Sequence[Metric]) -> MetricExportResult:
+    def export(
+        self, metrics: Sequence[Metric], *args, **kwargs
+    ) -> MetricExportResult:
         for metric in metrics:
             self.out.write(self.formatter(metric))
         self.out.flush()
         return MetricExportResult.SUCCESS
 
-    def shutdown(self) -> None:
+    def shutdown(self, *args, **kwargs) -> None:
```

Comment:
Should this method have a timeout argument?

Reply:
I elided it here because it wouldn't be used (would be captured by *args, **kwargs).